README.rst

```diff
@@ -42,8 +42,8 @@ Initial setup and installation:
 
 - **recommended:** create and activate a python 3.6 virtualenv::
 
-     virtualenv ppa -p python3.6
-     source ppa/bin/activate
+     virtualenv venv -p python3.6
```

Comment:
the venv "ppa" placed lib/bin into the django directory ppa. This caused many problems.

Reply:
I see! The instructions don't specify where to run this command, and you - quite logically! - ran it inside the git checkout directory. That would definitely be problematic. I'm not sure it's actually a good idea to have the virtualenv inside the project directory, though - it seems like it could cause problems.  @thatbudakguy where do you put yours? 

`venv` might not be great because that's the name of the module in python3; I could see using `env`, though if we want something generic. 

I also think we should also update this to use a different command, so we don't require installing `virtualenv`:
`python3 -m venv ppa -p python3.6`

Reply:
I always put mine in the project directory, fwiw. I also always name them `venv` so that my alias for `source venv/bin/activate` works in any folder.

Reply:
And I ended up using anaconda. We could just say "create and activate a python virtual environment with Python 3.6"

Reply:
🤔 I think I'm ok with that. That seems better than explicit directions that can cause problems, and probably to set this up you need some experience with python anyway.